projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-bar/class-wpcom-admin-bar.php

```diff
@@ -55,7 +55,7 @@ class WPCOM_Admin_Bar extends \WP_Admin_Bar {
 	 * }
 	 */
 	public function add_node( $args ) {
-		if ( ! is_array( $args ) || empty( $args['href'] ) ) {
+		if ( ! is_array( $args ) || empty( $args['href'] ) || ! isset( $args['id'] ) ) {
```

Comment:
I think this changes more behavior than is intended. Why shouldn't the code on lines 74–80 still be run?

I'd add the check to line 63 instead, which is the start of the piece that was added in #39113.